07-forms/02-MeetupForm/components/MeetupForm.vue

```diff
@@ -1,162 +1,236 @@
 <template>
-  <form class="meetup-form">
+  <form class="meetup-form" @submit="submit">
     <div class="meetup-form__content">
       <fieldset class="meetup-form__section">
         <UiFormGroup label="Название">
-          <UiInput name="title" />
+          <UiInput name="title" v-model="title" />
         </UiFormGroup>
+
         <UiFormGroup label="Дата">
-          <UiInputDate type="date" name="date" />
+          <UiInputDate name="date" type="date" v-model="date" />
         </UiFormGroup>
+
         <UiFormGroup label="Место">
-          <UiInput name="place" />
+          <UiInput name="place" v-model="place" />
         </UiFormGroup>
+
         <UiFormGroup label="Описание">
-          <UiInput multiline rows="3" name="description" />
+          <UiInput multiline rows="3" name="description" v-model="description" />
         </UiFormGroup>
+
         <UiFormGroup label="Изображение">
-          <!--
-               Предлагается сохранять файл для будущей загрузки во временное поле imageToUpload
-               и передавать в объекте со всеми данными формы
-          -->
           <ui-image-uploader
             name="image"
-            :preview="meetup.image"
-            @select="meetup.imageToUpload = $event"
-            @remove="meetup.imageToUpload = null"
-          />
+            :preview="image"
+            @select="selectImage"
+            @remove="removeImage" />
         </UiFormGroup>
       </fieldset>
 
       <h3 class="meetup-form__agenda-title">Программа</h3>
-      <!--
+
       <meetup-agenda-item-form
-         :key="agendaItem.id"
-         :agenda-item="..."
-         class="meetup-form__agenda-item"
-       />
-       -->
+        class="meetup-form__agenda-item"
+        v-for="itm, idx in agenda"
+        :key="itm.id"
+        v-model:agendaItem="agenda[idx]"
+        @remove="() => removeAgendaItem(itm)" />
```

Comment:
Можно не оборачивать в функцию

```suggestion
        @remove="removeAgendaItem(itm)" />
```